src/notification/index.js

```diff
@@ -48,7 +48,7 @@ export const getAccountFromNotificationData = (
 
   const urlMatches = [];
   identities.forEach((account, i) => {
-    if (account.realm.href === realmUrl.href) {
+    if (account.realm.host === realmUrl.host) {
```

Comment:
We usually compare `.origin`, not `.host` (e.g., as in e47a14308). `.origin` should have everything in `.host`, plus the scheme, like `http` or `https`, which I'd be inclined to differentiate on. What do you think?

Reply:
Ah, yeah, I was considering that when I made this, but didn't realize `.origin` was a thing. I'll change that, since in theory someone could run two different zulip servers, one on `http://example.com` and the other on `https://example.com`, even if that would be... extremely chaotic :p